components/MoveList.js

```diff
@@ -18,8 +20,11 @@ export default class MoveList extends React.Component {
         key={JSON.stringify(movelist)}/>
     ));
 
+    const renderedMechanics = !!mechanics && <Mechanics mechanicsTitle="Mechanics" mechanics={mechanics}/>
```

Comment:
I think it would be okay to assume that the title for this section will always be "Mechanics" so I'm not sure we get anything from passing it in as a variable.